conf/nginx.conf

```diff
@@ -70,4 +70,6 @@ server {
     rewrite ^/proof-of-work/proof-of-work/?$                             $scheme://$http_host/proof-of-work/overview permanent;
     rewrite ^/governance/introduction-to-decred-governance/?$            $scheme://$http_host/governance/overview permanent;
     rewrite ^/contributing/contributing-to-decred/?$                     $scheme://$http_host/contributing/overview permanent;
+    rewrite ^/contributing/using-github/?$                               https://devdocs.decred.org/contributing/contributor-guidelines/ permanent;
+    rewrite ^/contributing/source-code-contributions/?$                  https://devdocs.decred.org/contributing/ permanent;
```

Comment:
https://devdocs.decred.org/contributing/ returns a 403

Both of these redirects need to be towards https://devdocs.decred.org/contributing/contributor-guidelines/